src/view/filter_mode/scene_tab.py

```diff
@@ -0,0 +1,155 @@
+"""Widget containing a nodeeditor for one scene."""
+import logging
+
+from PySide6.QtWidgets import QWidget, QGridLayout
+
+from pyqtgraph.flowchart.Flowchart import Flowchart, Terminal
+from pyqtgraph.flowchart.library import NodeLibrary
+
+from model.board_configuration import Scene, Filter, UniverseFilter
+
+from . import nodes
+
+class SceneTabWidget(QWidget):
+    """Widget containing a nodeeditor for one scene"""
+    # To handle loading from file.
+    _type_to_node: dict[int, type] = {
+        0: nodes.Constants8BitNode.nodeName,
```

Comment:
Again, this is information that could be deduced from the filter data, given a separate module that enumerates them.